project.yml

```diff
@@ -0,0 +1,99 @@
+    NSE_PROVISIONING_PROFILE_SPECIFIER: "Vector NSE: App Store"
+    SHARE_EXTENSION_PROVISIONING_PROFILE: 1a3be143-50c7-4ae2-834e-00596a053141
+    SHARE_EXTENSION_PROVISIONING_PROFILE_SPECIFIER: "Vector Share Extension: App Store"
+    SIRI_INTENTS_PROVISIONING_PROFILE: 18a66f93-ffe1-4008-b343-58350cc65023
+    SIRI_INTENTS_PROVISIONING_PROFILE_SPECIFIER: "Vector Siri Intents: App Store"
+
+settings:
+  base:
+    IPHONEOS_DEPLOYMENT_TARGET: 11.0
+    MARKETING_VERSION: 1.1.3
+    CURRENT_PROJECT_VERSION: 1.1.3
+    SDKROOT: iphoneos
+    TARGETED_DEVICE_FAMILY: 1,2
+    ALWAYS_SEARCH_USER_PATHS: NO
+    CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED: YES
+    CLANG_CXX_LANGUAGE_STANDARD: gnu++0x
+    CLANG_CXX_LIBRARY: libc++
+    CLANG_ENABLE_MODULES: YES
+    CLANG_ENABLE_OBJC_ARC: YES
+    CLANG_WARN__DUPLICATE_METHOD_MATCH: YES
+    CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING: YES
+    CLANG_WARN_BOOL_CONVERSION: YES
+    CLANG_WARN_COMMA: YES
+    CLANG_WARN_CONSTANT_CONVERSION: YES
+    CLANG_WARN_DEPRECATED_OBJC_IMPLEMENTATIONS: YES
+    CLANG_WARN_DIRECT_OBJC_ISA_USAGE: YES_ERROR
+    CLANG_WARN_EMPTY_BODY: YES
+    CLANG_WARN_ENUM_CONVERSION: YES
+    CLANG_WARN_INFINITE_RECURSION: YES
+    CLANG_WARN_INT_CONVERSION: YES
+    CLANG_WARN_NON_LITERAL_NULL_CONVERSION: YES
+    CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF: YES
+    CLANG_WARN_OBJC_LITERAL_CONVERSION: YES
+    CLANG_WARN_OBJC_ROOT_CLASS: YES_ERROR
+    CLANG_WARN_RANGE_LOOP_ANALYSIS: YES
+    CLANG_WARN_STRICT_PROTOTYPES: YES
+    CLANG_WARN_SUSPICIOUS_MOVE: YES
+    CLANG_WARN_UNREACHABLE_CODE: YES
+    DEFINES_MODULE: YES
+    ENABLE_STRICT_OBJC_MSGSEND: YES
+    GCC_C_LANGUAGE_STANDARD: gnu99
+    GCC_NO_COMMON_BLOCKS: YES
+    GCC_WARN_64_TO_32_BIT_CONVERSION: YES
+    GCC_WARN_ABOUT_RETURN_TYPE: YES_ERROR
+    GCC_WARN_UNDECLARED_SELECTOR: YES
+    GCC_WARN_UNINITIALIZED_AUTOS: YES_AGGRESSIVE
+    GCC_WARN_UNUSED_FUNCTION: YES
+    GCC_WARN_UNUSED_VARIABLE: YES
+  debug:
+    CODE_SIGN_IDENTITY: iPhone Developer
+    CODE_SIGN_IDENTITY[sdk=iphoneos*]: iPhone Developer
+    COPY_PHASE_STRIP: NO
+    ENABLE_TESTABILITY: YES
+    GCC_DYNAMIC_NO_PIC: NO
+    GCC_OPTIMIZATION_LEVEL: 0
+    GCC_PREPROCESSOR_DEFINITIONS: DEBUG=1 $(inherited)
+    GCC_SYMBOLS_PRIVATE_EXTERN: NO
+    MTL_ENABLE_DEBUG_INFO: YES
+    ONLY_ACTIVE_ARCH: YES
+  release:
+    CODE_SIGN_IDENTITY: iPhone Distribution
+    CODE_SIGN_IDENTITY[sdk=iphoneos*]: iPhone Distribution
+    COPY_PHASE_STRIP: YES
+    ENABLE_NS_ASSERTIONS: NO
+    MTL_ENABLE_DEBUG_INFO: NO
+    SWIFT_COMPILATION_MODE: wholemodule
+    VALIDATE_PRODUCT: YES
+options:
+  defaultConfig: Release
+  groupSortPosition: bottom
+  transitivelyLinkDependencies: false
+  createIntermediateGroups: true
+  postGenCommand: sh createWorkspace.sh
+include:
+  - path: Targets/Riot.yml
+  - path: Targets/RiotTests.yml
+  - path: Targets/RiotShareExtension.yml
+  - path: Targets/SiriIntents.yml
+  - path: Targets/RiotNSE.yml
```

Comment:
I think we can keep the app target name as is but we can now use generic terms at configuration level:
- `Riot.yml` -> `Application.yml`
- `RiotTests.yml` -> `ApplicationTests.yml`
- `RiotShareExtension.yml` -> `ShareExtension.yml`
- `RiotNSE.yml` -> `NotificationServiceExtension.yml`
- `SiriIntents.yml` -> Keep this name